src/hotspot/share/prims/jvmtiEnvBase.cpp

```diff
@@ -864,6 +864,7 @@ JvmtiEnvBase::get_subgroups(JavaThread* current_thread, Handle group_hdl, jint *
 
   // This call collects the strong and weak groups
   JavaThread* THREAD = current_thread;
+  JvmtiJavaUpcallMark jjum(current_thread);
```

Comment:
Add comment like you did above for the JvmtiEnv::InterruptThread case.

Reply:
Okay, thanks! Added now.